sample/build.gradle

```diff
@@ -17,6 +17,9 @@ android {
   }
 
   buildTypes {
+    debug {
```

Comment:
Hmmm. Not 100% sure I need this here. Will try a fresh build without this config option. I'm assuming we really only care about the coverage for the actual library vs the sample application?